src/topology/category/Profinite.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2020 Kevin Buzzard. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Buzzard
+-/
+
+import topology.category.CompHaus
+
+/-!
+# The category of Profinite Types
+We construct the category of profinite topological spaces,
+often called profinite sets -- perhaps they could be called
+profinite types in Lean.
```

Comment:
It might be nice to include in the docstring that you're defining them as compact Hausdorff totally disconnected spaces rather than as a projective limit of finite sets

Reply:
I beefed up the docsting.

Reply:
Looks great, thanks!